build/source/noah-mp/module_sf_noahmplsm.F

```diff
@@ -63,6 +64,9 @@ module noahmp_globals
   REAL    :: RSMAX        !maximum stomatal resistance ( Assigned in REDPRM )
   REAL    :: TOPT         !optimum transpiration air temperature.
 
+  ! MPC change: make variables private for a given thread
+  !$omp threadprivate(NROOT, RGL, RSMIN, HS, RSMAX, TOPT)
+
```

Comment:
Suggest adding a stronger suggestion that at some point, these should be moved to a proper data structure.

Reply:
Good point. This suggestion will be made as part of CTSM development.
https://github.com/ESCOMP/ctsm